src/Model/ModelManager.php

```diff
@@ -363,6 +362,17 @@ public function getExportFields($class)
      */
     public function getModelInstance($class)
     {
+        $r = new \ReflectionClass($class);
```

Comment:
Instead to let the exception from the reflection API to bubble up, maybe we could thrown our own exception if the given class doesn't exist.